specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,235 @@
+ElementBackgroundGrid
+===
+
+# Background
+
+The `ElementBackgroundGrid` API in this spec is a Xaml Grid that lets you "draw" its background by
+pointing at another element; the rendering of that other element becomes the background of
+the Grid.
+
+`ElementBackgroundGrid` element is similar to several precedents:
+* [SwapChainBackgroundPanel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SwapChainBackgroundPanel),
+which is a `Grid` that renders its background using a swap chain.
+* [CompositionMaskBrush](https://docs.microsoft.com/uwp/api/Windows.UI.Composition.CompositionMaskBrush),
+which  is a brush that gets its content from another (source) brush.
+* WPF's [VisualBrush](https://docs.microsoft.com/dotnet/api/System.Windows.Media.VisualBrush),
+which uses an element's rendering to draw a brush.
+
+`ElementBackgroundGrid` is most similar to the WPF `VisualBrush`, the difference being that `ElementBackgroundGrid`
+is an element rather than a Brush. So it can't for example be rendered as text foreground.
+
+This new `ElementBackgroundGrid` will be used in the rendering of the 
+[DatePicker](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.DatePicker)
+control.
+
+# API Pages
+
+## ElementBackgroundGrid class
+
+A [Grid](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Grid)
+whose background can be the rendering of another element, or using another element
+as a mask.
+
+```csharp
+public class ElementBackgroundGrid : Grid
+{
+    public UIElement SourceElement { get; set; }
+    public Color ReplacementColor { get; set; }
+}
+```
+
+Set the `SourceElement` property to indicate which element's rendering should be drawn
+as the background of the `ElementBackgroundGrid`. If the `Background` property is set, it
+is rendered on top of the `SourceElement's` rendering.
```

Comment:
I believe this description is backwards, though i'm confused by the phrase "SourceElement's Rendering" -- are we talking about the rendering done by ElementBackgroundGrid? If so, then it goes: Background (bottom most), SourceElement (with color modifications), Children (on top).

Reply:
"If the `Background` property is set, the `SourceElement's` rendering is on top of it. (The source is z-ordered on top of the background.)"

Reply:
Recommend: fix